src/Aspire.Dashboard/Components/Pages/Traces.razor

```diff
@@ -61,6 +61,19 @@
                                 @($"+{another_name.ItemsOverflow.Count()}")
                             </span>
                         </MoreButtonTemplate>
+                        <OverflowTemplate Context="another_name">
```

Comment:
why do we use this weird name?

Reply:
I copied it from a sample in one place and have then been using it everywhere. Will update it at some point.